descholar-ceo/AnnounceIT
# #170864196 User get specific announcement api

#### What does this PR ​ do​ ?
As a user, I should be able to a specific announcement using annoucement-id on the API

#### Description of Task to be completed?
 * Write unit test for this api
 * Add function for retrieving a specific announcement, and all other setups

#### How should this be manually tested?
* Clone this repository in case you don't have it using this command `git clone https://github.com/descholar-ceo/AnnounceIT.git`
* Type these commands : 
    *  `cd AnnounceIT` to go in the root directory of the project
    * `npm install` to install all the packages
    * `touch .env` to create dotenv file which will keep privately sensitive data for you,
    * Open the dotenv created above and copy paste this `PORT=8080` press Enter and copy paste 
     this `TOKEN_KEY=descholar` __you can always modify this data by changing the values of 
     PORT and TOKEN_KEY as you wish,__
    * `npm start` to start the server, sothat the api will be up and running
    * Open [Postman](https://www.getpostman.com/) and create a get request to this link `http://localhost:8080/api/v1/announcements/get-specific-announcement/1`,  
you will see that this announcement is not yet available, to add an item to do more test please refer to 
 [This PR](https://github.com/descholar-ceo/AnnounceIT/pull/20)

#### Any background context you want to provide?
* Use [Postman](https://www.getpostman.com/), to test it or any other rest-api testing tools available to you

#### What are the relevant pivotal tracker stories?
[170864196](https://www.pivotaltracker.com/n/projects/2429498#)

#### Screenshots (​ if​ appropriate)

#### Questions:
No Question
